bears/csv/CSVLintBear.py

```diff
@@ -17,6 +39,7 @@ class CSVLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/dlzd1y6vnwx7kk1a17p2gk7ll'
```

Comment:
I'll add a photo of why I had to do this. 
Maybe there's a better regex that I couldn't map.